### PR TITLE
DA Path and PHP Installation Fixes

### PR DESCRIPTION
**Pull request by Dala0**
Fixes path errors, installs setup dependencies for deployment, and fixes installation of PHP on the Docker image

@fredex42

### CODE REVIEW

File: packer/endpointserver.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ provisioners:
       - echo Installing PHP
       - echo ------------------------------------
 
-      - sudo yum -y --enablerepo=remi,remi-php72 install php phpunit7 php-common php-mysql php-fpm php-pecl-memcache php-pecl-memcached
+      - if [ -f /etc/fedora-release ]; then sudo yum -y --enablerepo=remi install php phpunit7 php-common php-mysql php-fpm php-pecl-memcache php-pecl-memcached; else sudo yum -y --enablerepo=remi,remi-php72 install php phpunit7 php-common php-mysql php-fpm php-pecl-memcache php-pecl-memcached; fi
```
Comment (fredex42) [inline packer/endpointserver.yaml]: i think that this belongs in a seperate PR?

File: .circleci/config.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ jobs:
       CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
       MYSQL_PWD: testuser
     docker:
-      - image: dala0/endpointbuild:6
+      - image: dala0/endpointbuild:7
```
Comment (fredex42) [inline .circleci/config.yml]: i think that this belongs in a seperate PR

File: .circleci/config.yml [hunk 3]
```diff
@@ -49,8 +49,8 @@ jobs:
           - /opt/vendor
     - run: source /etc/rc.d/init.d/functions && daemon --pidfile=/var/run/nginx.pid nginx -c /etc/nginx/nginx.conf
     - run: mkdir /var/run/php-fpm
-    - run: cp /root/multimedia-encoding-endpoint/endpoint/video.php /var/www/
-    - run: cp /root/multimedia-encoding-endpoint/endpoint/common.php /var/www/
+    - run: cp /root/multimedia-encoding-endpoint/endpoint/video.php /var/www/html/
```
Comment (fredex42) [inline .circleci/config.yml]: likewise this is a seprate PR

File: packer/endpointserver.yaml (hunk 1 above)
Comment (Dala0) [reply packer/endpointserver.yaml]: Why? The code in the master does not work.

**Review by fredex42 — APPROVED** [review]
👍

**Pull request merged**